### DISCUSSION TITLE
Error of "Face_spoof_detection_img"

### DISCUSSION THREAD

**Discussion by a5111295** [Ignitarium-Renesas/RZV2L_AiLibrary#9] [General]
Hello ,
I used "Face_spoof_detection_img" of  "06-Face_recognition_spoof_detection
spoof_detection".
but there is error , when setting image size of not sample.bmp
-----
[ERROR] Failed to run fread():error=0
[ERROR] Failed to read BMP file: sample.bmp

Could  you please fix it ?

**Comment by a5111295** [comment]
I moved this comment to "issues #13"

**Reply by mohitb-ign** [reply]
Answered in issues section. Please have a look.
